pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/NamespacesBase.java

```diff
@@ -3000,6 +3001,47 @@ protected void internalSetOffloadPolicies(AsyncResponse asyncResponse, OffloadPo
         }
     }
 
+    protected void internalRemoveOffloadPolicies(AsyncResponse asyncResponse) {
+        validateNamespacePolicyOperation(namespaceName, PolicyName.OFFLOAD, PolicyOperation.WRITE);
+        validatePoliciesReadOnlyAccess();
+
+        try {
+            Stat nodeStat = new Stat();
+            final String path = path(POLICIES, namespaceName.toString());
+            byte[] content = globalZk().getData(path, null, nodeStat);
+            Policies policies = jsonMapper().readValue(content, Policies.class);
+
+            policies.offload_policies = null;
+            String updatedOffloadPolicies = jsonMapper().writeValueAsString(policies.offload_policies);
```

Comment:
Maybe this is not necessary? The log could be just like Successfully remove the namespace offload policies.